tsfresh/transformers/feature_selector.py

```diff
@@ -82,7 +84,7 @@ def fit(self, X, y):
             y = pd.Series(y.copy())
 
         df_bh = check_fs_sig_bh(X, y, self.settings)
-        self.relevant_features = df_bh.loc[df_bh.rejected].Feature
+        self.relevant_features = df_bh.loc[df_bh.rejected].p_value
```

Comment:
Is there a reason for dismissing the relation between feature names and p_values/significance?